### PR TITLE
MWPW-109802 - [SEO Structured Data] Breadcrumb

### PR DESCRIPTION
**Pull request by jckautzmann**
* adds JSON-LD data describing the breadcrumb in the head tag

Resolves: [MWPW-109802](https://jira.corp.adobe.com/browse/MWPW-109802)

**Test URLs:**
- Before: https://main--milo--adobecom.hlx.page/drafts/jck/test1?martech=off
- After: https://breadcrumb-seo--milo--adobecom.hlx.page/drafts/jck/test1?martech=off

### CODE REVIEW

File: libs/blocks/gnav/gnav.js [hunk 1]
```diff
@@ -448,6 +449,28 @@ class Gnav {
     }
   };
 
+  setBreadcrumbSEO = () => {
+    const breadcrumb = this.el.querySelector('.breadcrumbs');
+    if (breadcrumb) {
+      const isSeo = breadcrumb.classList.contains('seo');
```
Comment (fullcolorcoder) [inline libs/blocks/gnav/gnav.js:455]: Was there author discussion around this? I'd think that breadcrumbs would be an always generating JSON-LD type thing.

**Comment by auniverseaway** [comment]
@jckautzmann this is awesome! I thought about this and I think @fullcolorcoder is on to something.

I talked with SEO about this and here's the final piece: It should be **_opt-out_** and the check should be something like:

```js
if (getMetadata('breadcrumb-seo') !== 'off')
```

This gets us inheritance through metadata.xlsx as well as being able to opt out via the metadata block.

**Review by auniverseaway — APPROVED** [review]
Small change the opt-in / opt-out code, otherwise: looks great!

**Comment by jckautzmann** [comment]
@auniverseaway Thx for clarifying. I addressed the opt-out option with https://github.com/adobecom/milo/pull/276/commits/8c56a2ef09464fe0ec5e617e113115b7d71c4476.

File: libs/blocks/gnav/gnav.js [hunk 2]
```diff
@@ -448,6 +449,28 @@ class Gnav {
     }
   };
 
+  setBreadcrumbSEO = () => {
+    const breadcrumb = this.el.querySelector('.breadcrumbs');
+    if (breadcrumb) {
```
Comment (chrischrischris) [inline libs/blocks/gnav/gnav.js:454]: nit: early return (also on L456)

File: libs/blocks/gnav/gnav.js (hunk 1 above)
Comment (jckautzmann) [reply libs/blocks/gnav/gnav.js]: @fullcolorcoder Thx for reviewing. I addressed the opt-out option based on @auniverseaway comment below with https://github.com/adobecom/milo/commit/8c56a2ef09464fe0ec5e617e113115b7d71c4476.

**Review by chrischrischris — APPROVED** [review]

**Pull request merged**